### PR TITLE
Added build script for requests-oauthlib

### PR DESCRIPTION
**Pull request by atharv-phadnis**
Package Name: **requests-oauthlib**
Package Version: **v1.3.0**
Package URL: [requests-oauthlib](https://github.com/requests/requests-oauthlib)

Tested on: **UBI 8.5**
Build Errors: **None**
Test Errors: **None**

### CODE REVIEW

**Comment by seth-priya** [comment]
@Ryan-Dmello please review

File: r/requests-oauthlib/requests-oauthlib_ubi_8.5.sh [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+		echo "$PACKAGE_URL $PACKAGE_NAME"
+		echo "$PACKAGE_NAME  |  $PACKAGE_URL |  $PACKAGE_VERSION | $OS_NAME | Fail |  Clone_Fails"
+    	exit 0
+fi
+
+cd $HOME_DIR/$PACKAGE_NAME
+git checkout $PACKAGE_VERSION
+
+if ! tox -e py36; then
+	echo "------------------$PACKAGE_NAME:install_or_test_fails---------------------"
```
Comment (Ryan-Dmello) [inline r/requests-oauthlib/requests-oauthlib_ubi_8.5.sh:40]: Tox command is going to perform package testing only. Please refer file tox.ini for details.

For installation use following command, after line 37:
1. pip3 install ./

Please update the wordings, after adding installation command.

File: r/requests-oauthlib/requests-oauthlib_ubi_8.5.sh [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+	echo  "$PACKAGE_URL $PACKAGE_NAME " 
+	echo "$PACKAGE_NAME  |  $PACKAGE_URL | $PACKAGE_VERSION | $OS_NAME | Fail |  Install_or_test_Fails"
```
Comment (Ryan-Dmello) [inline r/requests-oauthlib/requests-oauthlib_ubi_8.5.sh:42]: Please update the wordings, after adding installation command.

**Review by Ryan-Dmello — CHANGES_REQUESTED** [review]
@atharv-phadnis Changes other than mentioned comments looks ok to me.

File: r/requests-oauthlib/requests-oauthlib_ubi_8.5.sh (hunk 2 above)
Comment (atharv-phadnis) [reply r/requests-oauthlib/requests-oauthlib_ubi_8.5.sh]: Made the necessary changes, please review.

**Comment by Ryan-Dmello** [comment]
Latest changes looks ok

**Review by Ryan-Dmello — APPROVED** [review]

**Pull request merged**